easybuild/easyconfigs/i/intel/intel-2019b.eb

```diff
@@ -0,0 +1,21 @@
+easyblock = 'Toolchain'
+
+name = 'intel'
+version = '2019b'
+
+homepage = 'https://easybuild.readthedocs.io/en/master/Common-toolchains.html#intel-toolchain'
+description = "Compiler toolchain including Intel compilers, Intel MPI and Intel Math Kernel Library (MKL)."
+
+toolchain = SYSTEM
+
+local_compver = '2019.4.243'
+local_gccver = '8.3.0'
+dependencies = [
+    ('GCCcore', local_gccver),
+    ('binutils', '2.32', '-GCCcore-%s' % local_gccver),
+    ('iccifort', local_compver),
+    ('impi', '2019.4.243', '', ('iccifort', local_compver)),
```

Comment:
While building the easyconfigs using `intel/2018b` with the current `intel/2019b` proposal, I ran into a couple of segfault issues like:

```
/software/impi/2019.4.243-iccifort-2019.4.243/intel64/bin/mpirun: line 103: 147382 Segmentation fault      mpiexec.hydra "$@" 0<&0
```

(this one was with `FFTW-3.3.8-intel-2019b.eb` when the MPI tests were run)

This seems to be a known problem with Intel MPI 2019 update 4 when it's restricted CPU-wise by cgroups, cfr. https://it.stonybrook.edu/help/kb/intel-mpi-segmentation-faults.

@bartoldeman @damianam @ocaisa Does this ring a bell for you? Should we (still) stick to the latest 2018 (update 5 currently)?!

Reply:
Same issue with `YAXT-0.6.0-intel-2019b.eb`, configure fails with:
```
configure: error: failed to run MPI programs
```

`config.log` confirms it's due to a similar `Segmentation fault`